### ISSUE TITLE
Failure when installing Verrazzano on OLCNE

### ISSUE THREAD

**Issue by rafabene** [verrazzano/verrazzano#262]
When installing Verrazzano using the first script  that installs Istio, I receive the following message:

> [2020-09-09 16:41:21 UTC] The Service "grafana" is invalid: spec.ports[0].nodePort: Forbidden: may not be used when `type` is 'ClusterIP'
[2020-09-09 16:41:21 UTC] Installing Istio                                                       [FAILED]
[2020-09-09 16:41:21 UTC]

**Comment by mgianatagh** [comment]
I have asked if someone from the team that is familiar with OLCNE can help.

**Comment by kminder** [comment]
@rafabene < Can you provide a bit more of the log before the 'The Service "grafana" is invalid' line.  Also what environment have you installed OLCNE on (e.g OCI)?

**Comment by rafabene** [comment]
I'll try to run again to get more logging info. I've used OLCNE through the vagrant box available here: https://github.com/oracle/vagrant-projects/tree/master/OLCNE

**Comment by mark-au** [comment]
Have you made any customisations to the OLCNE vagrant-projects? Did you install the Istio module of OLCNE before installing Verrazzano? 
Based on the error provided I guess you already have Istio installed and the Verrazzano is unable to configure it

**Comment by rafabene** [comment]
Yes. I did some modifications to OLCNE. Let me try the master branch.

**Comment by mark-au** [comment]
Make sure you don't have `DEPLOY_ISTIO=true` in the `.env.local`, as mentioned earlier I think that's the cause of your issue

**Comment by rafabene** [comment]
I tried to install without ISTIO and now I've stopped at the script 3 ( 3-install-verrazzano.sh ). The error log says: 

> 2020-09-21 16:58:09 UTC] Waiting for Rancher TLS cert to reach ready state
[2020-09-21 16:58:09 UTC] certificate.cert-manager.io/tls-rancher-ingress condition met
[2020-09-21 16:58:09 UTC] Waiting for ingress rancher in namespace cattle-system to have an IP
[2020-09-21 16:58:09 UTC] jq: error (at <stdin>:56): Cannot iterate over null (null)
[2020-09-21 16:58:15 UTC] jq: error (at <stdin>:56): Cannot iterate over null (null)
[2020-09-21 16:58:20 UTC] jq: error (at <stdin>:56): Cannot iterate over null (null)
[2020-09-21 16:58:25 UTC] jq: error (at <stdin>:56): Cannot iterate over null (null)
[2020-09-21 16:58:30 UTC] jq: error (at <stdin>:56): Cannot iterate over null (null)
[2020-09-21 16:58:35 UTC] jq: error (at <stdin>:56): Cannot iterate over null (null)
[2020-09-21 16:58:40 UTC] jq: error (at <stdin>:56): Cannot iterate over null (null)
[2020-09-21 16:58:45 UTC] jq: error (at <stdin>:56): Cannot iterate over null (null)
[2020-09-21 16:58:50 UTC] jq: error (at <stdin>:56): Cannot iterate over null (null)
[2020-09-21 16:58:55 UTC] jq: error (at <stdin>:56): Cannot iterate over null (null)
[2020-09-21 16:59:00 UTC] An error occurred - ingress rancher in namespace cattle-system did not have an IP address
[2020-09-21 16:59:00 UTC] Installing Verrazzano system components                                [FAILED]
[2020-09-21 16:59:00 UTC]
[2020-09-21 16:59:00 UTC] A failure occurred. Exiting with code 1.
